eng/service.proj

```diff
@@ -35,4 +35,13 @@
              SkipNonexistentTargets="true" />
   </Target>
 
+  <!--Run Aggregate Updates to Source -->
+  <Target Name="AggregateUpdatesToSource" AfterTargets="Build" Condition="'$(UpdateSource)' == 'true' or '$(AZURE_DEV_UPDATESOURCES)' == 'true'" >
```

Comment:
While it might not matter yet since there is only this one condition I suggest we set the UpdateSource property based on the AZURE_DEV_UPDATESOURCES env variable assuming it isn't already set. 

```
<PropertyGroup>
  <UpdateSource Condition="'$(UpdateSource)' == ''">$(AZURE_DEV_UPDATESOURCES)<UpdateSource>
</PropertyGroup>
```

Reply:
Can we have a more expressive name? UpdateSourceOnBuild?